### DISCUSSION TITLE
Change docs for the "Font Colors" in "Theme Overrides" > "Colors".

### DISCUSSION THREAD

**Discussion by PurrfectDreamm** [godotengine/godot-proposals#10695] [GUI]
In short: Change documentation for Font Colors along the path Theme Overrides > Colors > Font Colors, and add there information about how to actually change font color -> $label.add_theme_color_override("font_color", Color(1,1,1,1))
##Description: Where 1,1,1,1 - red, green, blue, alpha / r,g,b,a, and this need to be float numbers from 0.0 to 1.0

In details:
I'm new to Godot and game development in general, and I learning by creating various game mechanics, I decided to change the color of the text when the player takes damage (my idea was to make hp text color red and then after 0.25 sec set hp text color to green, I set up timer for this and tried to change the font_color...) I hovered over "Font Color" along the path Theme Overrides > Colors > Font Color, and as prompt said I tried to change the color of the text like this: %HP.font_color = red 
##red = Color(1, 0, 0, 1)
it didn't work... (before you ask, yes my HP label is set to unique name)
Then I tried this: %HP.set_font_color(red), that didn't work either. Then I decided to open the documentation for this Font Color and saw the same font_color [default: Color(1, 1, 1, 1)] - which I have already tried to change a hundred times and it does not work.
Then I started googling and on some forum I found a working method: 
$label.add_theme_color_override("font_color", Color(1,1,1,1))
Point is that this documentation for Font Color along the Theme Overrides path should be changed/revised, or at least just add additional information to this documentation, something like: "To set the text color, use add_theme_color_override("font_color", Color(r,g,b,a)", or something like that, idk, I just want to this to be like other helpful docs that I see and instantly understand how something works and how to change it in some way.
I am sure that this change would save many beginners time and nerves in the future, because as for now if you don't know how to set font_color for your text it's just impossible for you to learn how to do it via corresponding Font Color documentation.

Also I would like the option to change color smoothly rather than instantly, but that’s a completely different conversation.
And also would be great to have more usage examples for methods, properties, functions in docs.

**Comment by aXu-AP** [comment]
Theming is explained on Control's doc page, but I agree, many beginners likely don't know to look it in there. Adding tutorial bit in font color's description is not gonna cut it however, since the user might want to change some other themed property.

Maybe a little text could be added under the Theme Properties subtitle?
![kuva](https://github.com/user-attachments/assets/aaf6ea33-0260-484a-b10d-8e7b19d42129)

A side note: for your smooth color changing see [tweening](https://docs.godotengine.org/en/stable/classes/class_tween.html).

**Reply by PurrfectDreamm** [reply]
In short: I thank you for your answer. And if I come across something else where the docs don’t have what I’m looking for, then I’ll move up the node hierarchy and look for what I need, or maybe I'll just ask GPT4. But I believe there is still plenty thing that needs to be added for beginners, including the tip for Font Color changing, and make it as easy accessible as for position or scale, you hover over position and you immediately see exactly how to change it via code, and if you open docs for position, you can see more info how to manipulate it via code, and as for now it's not the case for Font Color (and docs have 0 value for Font Color because the things that docs say simply doesn't work in code).

I think this link to Control's themed properties as you showed on this image might help, but in my case only IF person spots it and clicks on it and searches specifically for $label.add_theme_color_override("font_color", Color(1,1,1,1)), which is a pretty low chance tbh, but for sure it's much better with this link to Control than without it!), and anyway I believe that it would be just better to add one simple line in that Theme Override's Font Color that says something like "If you want to change color, do/use that..." or a link to that add_theme_color_override in Control docs. Idk maybe I'm being picky but I really like how docs and tips are implemented in the Godot, it helped me a lot, and docs give me ability to figure out everything myself without need to google something or watch guides, and this was first time when I couldn't get value from docs (just because I didn't knew that Control docs have info about Themes Overrides methods and other stuff), and when I found solution on the internet, I thought "Whaaat? How could I ever come up with this devious line of code if there is nothing about it in the font color documentation?!". It just made me a little bit sad because I couldn't find help in Theme Override's Font Color docs for simple color change.

**Comment by Mickeon** [comment]
All Control & co.'s class reference is in a pretty rough spot, in my opinion. Very inconsistent and general awkward wording make it a confusing mess to navigate.
